packages/medusa/src/utils/flag-router.ts

```diff
@@ -19,13 +20,16 @@ export class FlagRouter implements FeatureFlagTypes.IFlagRouter {
    * @return {boolean} - Whether the flag is enabled or not
    */
   public isFeatureEnabled(flag: string | Record<string, string>): boolean {
-    if (typeof flag === `string`) {
+    if (isString(flag)) {
       return !!this.flags[flag]
     }
 
-    const [nestedFlag, value] = Object.entries(flag)[0]
+    if (isObject(flag)) {
+      const [nestedFlag, value] = Object.entries(flag)[0]
+      return !!this.flags[nestedFlag]?.[value]
+    }
 
-    return !!this.flags[nestedFlag]?.[value]
+    throw Error("Argument to `isFeatureEnabled` must be a string or an object")
```

Comment:
suggestion: `'flag' must be a string or an object` I am not sure the name of the method is necessary since it will be in the stack trace right?